lib/number.js

```diff
@@ -53,12 +53,29 @@ internals.Number = class extends Any {
 
     multiple(base) {
 
-        Hoek.assert(Hoek.isInteger(base), 'multiple must be an integer');
```

Comment:
So that means:

- The developer should first validate if the multiple is zero (the developer is using Joi for validation and there is no way to validate that in Joi).
- Allowing floats has no meaning, because it will throw exception anyways.

Am I right?
So in effect we are just making the changes for allowing refs.?